src/scripts/activities/activities.component.js

```diff
@@ -0,0 +1,145 @@
+import _ from 'underscore';
+
+import template from './activities.html';
+import './activities.scss';
+
+function ActivitiesController($scope, dimStoreService, dimDefinitions, dimSettingsService, $translate) {
+  'ngInject';
+
+  const vm = this;
+
+  vm.settings = dimSettingsService;
+
+  // TODO: it's time for a directive
+  vm.toggleSection = function(id) {
+    vm.settings.collapsedSections[id] = !vm.settings.collapsedSections[id];
+    vm.settings.save();
+  };
+
+  vm.settingsChanged = function() {
+    vm.settings.save();
+  };
+
+  // TODO: Ideally there would be an Advisors service that would
+  // lazily load advisor info, and we'd get that info
+  // here. Unfortunately we're also using advisor info to populate
+  // extra info in Trials cards in Store service, and it's more
+  // efficient to just fish the info out of there.
+
+  // TODO: it'll be nice to replace this pattern with RxJS observables
+  function init(stores = dimStoreService.getStores()) {
+    if (_.isEmpty(stores)) {
+      return;
+    }
+
+    stores = stores.filter((s) => s.id !== 'vault');
+
+    const whitelist = [
+      'vaultofglass',
+      'crota',
+      'kingsfall',
+      'wrathofthemachine',
+      //      'elderchallenge',
+      'nightfall',
+      'heroicstrike',
+    ];
+
+    dimDefinitions.getDefinitions().then((defs) => {
+      const rawActivities = stores[0].advisors.activities;
+      vm.activities = _.chain(rawActivities)
+        .filter('activityTiers') // exclude trials, iron banner, armsday, xur
+        .filter((a) => {
+          return whitelist.includes(a.identifier);
+        })
+        .sortBy((a) => {
+          const ix = whitelist.indexOf(a.identifier);
+          return (ix === -1) ? 999 : ix;
+          //          return -a.display.categoryHash;
+        })
+        .map((a) => processActivities(defs, stores, a))
+        .value();
```

Comment:
Just a heads up, I'm going to be removing usages of `chain` in hopes of maintaining future compatibility with Lodash (which has chain, but is somewhat inefficient bundle-size-wise with it).